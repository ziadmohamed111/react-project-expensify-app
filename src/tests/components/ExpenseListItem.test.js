import React from "react"
import { shallow } from "enzyme";
import ExpenseListItem from "../../components/ExpenseListItem"
import expenses from "../fixtures/expenses"

test("sholud render Expense list item with one expense", ()=>{
    const expenseitem = expenses[1]
    const wrapper = shallow( <ExpenseListItem {...expenseitem}/>)
    expect(wrapper).toMatchSnapshot()
})