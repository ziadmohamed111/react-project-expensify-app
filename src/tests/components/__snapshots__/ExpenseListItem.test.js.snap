// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sholud render Expense list item with one expense 1`] = `
<div>
  <[object Object]
    to="/edit/2"
  >
    <h3>
       
      Rent
       
    </h3>
  </[object Object]>
  <p>
     $
    100000
     - 
    -345600000
     
  </p>
</div>
`;
